prog/vm/host_nexus.rb

```diff
@@ -109,11 +109,15 @@ def self.assemble(sshable_hostname, location: "hetzner-hel1", net6: nil, ndp_nee
       vm.update(display_state: "rebooting")
     }
 
-    sshable.cmd("sudo systemctl reboot")
-
     decr_reboot
 
-    hop_wait_reboot
+    begin
+      sshable.cmd("sudo reboot")
+    rescue Net::SSH::Disconnect, Errno::ECONNRESET
+      hop_wait_reboot
```

Comment:
This raised "IOError: closed stream" error on my local dev. Is it inherited from one of these exception?

Reply:
Not sure, I was just testing this to see if this is enough. I'll send for review in few minutes.

Reply:
I added the IOError and it passed the E2E tests locally.